src/Notification/Notification.module.scss

```diff
@@ -0,0 +1,107 @@
+@import '../mixins.scss';
+@import '../variables.scss';
+
+.wrapper {
+    @include text-normal;
+
+    border-radius: $border-radius;
+    padding: 14px 18px 16px 15px;
+    width: 320px;
+    margin-bottom: 15px;
+}
+
+.default {
+    color: var(--grey-dark-2);
+    background-color: var(--white);
+    fill: var(--black);
+
+    .title {
+        color: var(--black);
+    }
+
+    .closeIcon {
+        fill: var(--grey-dark);
+    }
+}
+
+.header {
+    display: flex;
+    align-items: center;
+}
+
+.closeIcon {
+    cursor: pointer;
+    width: 13px;
+    height: auto;
```

Comment:
1. `height: auto;` is not necessary;
2. Should be positioned absolutely at the top-right corner, otherwise, this might happen:
![image](https://user-images.githubusercontent.com/25135650/53962390-02894e80-40fc-11e9-9f3b-66ef14158129.png)